### PR TITLE
ES6

### PR DESCRIPTION
**Pull request by commuterjoy**
Move to using import, const, let and arrow functions.

This is part of a tidy to migrate it to node v4.

### CODE REVIEW

File: tests/server/transforms/cohortSpec.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ describe('Cohort', function () {
 			expect(cohorts.cohorts).to.deep.equal([]);
 			expect(cohorts.incumbent).to.be.undefined;
 			done()
-		});
+		}).catch(err);
```
Comment (matthew-andrews) [inline tests/server/transforms/cohortSpec.js]: i think with mocha you can just return the promise object and it'll do this sort of stuff for you

Comment (matthew-andrews) [inline tests/server/transforms/cohortSpec.js]:
```js
it('Users with no products are anonymous', () => {
  return cohort(e)
    .then(cohorts => {
      expect(cohorts.productCodes).to.be.undefined;
      expect(cohorts.cohorts).to.deep.equal([]);
      expect(cohorts.incumbent).to.be.undefined;
    });
});
```

Comment (commuterjoy) [inline tests/server/transforms/cohortSpec.js]: ooh, never knew that. cheers.

**Comment by ironsidevsquincy** [comment]
On the front page, we've whitelisted the transforms we want babel to perform (basically the things that aren't in node 4 yet)

**Comment by commuterjoy** [comment]
@ironsidevsquincy ok, thanks. will do similar.

**Pull request merged**